README.md

```diff
@@ -43,4 +43,7 @@ $ cd book-club
 $ npm install
 ```
 
- 
```

Comment:
Move the badges to the top of your file. Also, you haven't properly configured codeclimate yet. Look into the travis build log for possible errors and rectify them.

Reply:
<img width="1440" alt="Screenshot 2020-08-27 at 15 25 52" src="https://user-images.githubusercontent.com/53005190/91455249-c4489780-e879-11ea-8aca-af43bfa6080d.png">

This is the screenshot of my travis log

Reply:
The build log is empty

Reply:
Okay, I'd have a look.

Reply:
Is this a public or a private repo? Can you switch over to [travis-ci](https://travis-ci.org). Add the project there and trigger a build.